### PR TITLE
Pesto app

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: templates/pesto/data/recipes.json [hunk 1]
```diff
@@ -0,0 +1,940 @@
+{
+  "recipes" :[
+    {
+      "title": "Cake",
+      "author": "Johnny Appleseed",
+      "description": "Delicious cake will be offered at the end of the test. It is not a lie.",
+      "imageUrl": "/components/app-layout/templates/pesto/img/img1.jpg",
```
Comment (frankiefu) [inline templates/pesto/data/recipes.json]: The path should be relative to `app-layout`, e.g. `img/img1.jpg`.  Same goes to the other paths in this file and the `background-image` in `recipe-list.html` (https://github.com/PolymerLabs/app-layout/pull/22/files#diff-41a0fae865f33fe1cf658e543e7afd33R33)

**Comment by keanulee** [comment]
Going to land this so I can work directly on header

**Comment by abdonrd** [comment]
This PR have a lot of changes. In short, what are the changes?

**Comment by keanulee** [comment]
This PR was merged into another development branch (`header`), so we didn't conduct a full code review (which will be done when this is merged to `master`). Most changes at this point are experimental, but mainly center around being able to build certain Material Design layouts as demonstrated in "Pesto".

**Comment by abdonrd** [comment]
Oh, you are right! I assumed that was merged into `master`. :sweat_smile: 

Sorry, but what is "Pesto"?

**Comment by keanulee** [comment]
It is a fictitious recipe app, kind of like the Zuperkulblog (in templates/publishing). We're more interested about the layout, but it's helpful to have a concept to center our discussions around.

**Comment by abdonrd** [comment]
Thanks @keanulee!

**Pull request merged**